documentation/shared/attributes.adoc

```diff
@@ -58,6 +58,7 @@
 
 // External links
 :aws-ebs: link:https://aws.amazon.com/ebs/[Amazon Elastic Block Store (EBS)^]
+:cluster-docs: link:https://kubernetes.io/docs/home/
```

Comment:
Maybe calling it `kubernetes-docs` would be more self-explaining?

Reply:
I went for generic for downstream. But I don't suppose it matters, so updated.